tools/Azure.Mcp.Tools.KeyVault/src/Commands/Certificate/CertificateImportCommand.cs

```diff
@@ -32,11 +32,9 @@ public sealed class CertificateImportCommand(ILogger<CertificateImportCommand> l
 
     public override string Description =>
         """
-        Imports (uploads) an existing certificate (PFX or PEM with private key) into an Azure Key Vault without generating
-        a new certificate or key material. This command accepts either a file path to a PFX/PEM file, a base64 encoded PFX,
-        or raw PEM text starting with -----BEGIN. If the certificate is a password-protected PFX, a password must be provided.
-        Returns certificate details including name, id, keyId, secretId, cer (base64), thumbprint, validity, and policy
-        subject/issuer.
+        Import an existing certificate (PFX or PEM with private key) into the vault. Accepts file path, base64 PFX, or raw
```

Comment:
Similar question here, do we need to say that what the permissions are? When we are already defining that to the client? Or is this an additional safeguard?